cognite/powerops/client/shop_result_files.py

```diff
@@ -89,6 +93,64 @@ def _parse_file(self, file: TextIO) -> dict:
     def file_content(self) -> str:
         return yaml.safe_dump(self.data, sort_keys=False)
 
+    def _prepare_plot_time_series(self, keys: Union[str, Sequence[str]]) -> dict:
+        if isinstance(keys, str):
+            keys = [keys]
+        return {key: self._retrieve_time_series_dict(key) for key in keys}
+
+    def _retrieve_time_series_dict(self, keys: str) -> dict[datetime, float]:
+        try:
+            data = self[keys]
+            assert isinstance(data, dict)
+            assert all(isinstance(key, datetime) for key in data.keys())
```

Comment:
Isn't `keys` actually a single key (so should be renamed to `key`). But this clashes with this other `key`...